LayoutTests/platform/mac-wk1/accessibility/roles-exposed-expected.txt

```diff
@@ -500,6 +500,16 @@ meter
       AXSubrole: AXMeter
       AXRoleDescription: level indicator
 
+menu
+      AXRole: AXList
+      AXSubrole: AXContentList
+      AXRoleDescription: content list
+
+li
+      AXRole: AXGroup
+      AXSubrole:
+      AXRoleDescription: group
+
```

Comment:
Why is this new output placed *after* the output of the meter element above? This does not match the order in `roles-exposed.html`